src/layer/annotation/index.ts

```diff
@@ -455,6 +465,9 @@ export class AnnotationUserLayer extends Base {
     this.annotationDisplayState.shaderControls.restoreState(
       specification[SHADER_CONTROLS_JSON_KEY],
     );
+    this.annotationDisplayState.color.restoreState(
```

Comment:
This is already restored by UserLayerWithAnnotationsMixin which is a base class.